### PR TITLE
small ui fixes for 3 column theme

### PR DESCRIPTION
**Pull request by janmichaelyu**
search results right margin
details for json bottom margin 
details for binaries bottom margin

* tested deploy on default and 3 column theme

### CODE REVIEW

**Comment by janmichaelyu** [comment]
@grtjn @jenbreese  Please review.

File: app/templates/ui/app/detail/detail.html [hunk 1]
```diff
@@ -8,13 +8,14 @@
           <pre ng-if="ctrl.type!=='json'" class="pre-xml">{{ctrl.xml}}</pre>
 
           <a ng-if="ctrl.type==='binary'" target="_blank" ng-href="/v1/documents?uri={{ctrl.uri}}&transform=download" class="btn btn-default">Download</a>
-
+          <p>&nbsp;</p>
```
Comment (grtjn) [inline app/templates/ui/app/detail/detail.html]: Can we fix this with CSS instead?

File: app/templates/ui/app/detail/detail.html [hunk 2]
```diff
@@ -8,13 +8,14 @@
...
         </div>
       </uib-tab>
       <uib-tab ng-if="ctrl.type!=='json'">
         <uib-tab-heading>JSON</uib-tab-heading>
         <div class="col-sm-9">
           <json-explorer data="ctrl.json"></json-explorer>
+          <p>&nbsp;</p>
```
Comment (grtjn) [inline app/templates/ui/app/detail/detail.html]: Can we fix this with CSS instead?

**Comment by grtjn** [comment]
@jenbreese Can you review the CSS?

File: app/templates/ui/app/detail/detail.html (hunk 2 above)
Comment (janmichaelyu) [inline app/templates/ui/app/detail/detail.html]: changed to CSS implementation

File: app/templates/ui/app/detail/detail.html (hunk 1 above)
Comment (janmichaelyu) [inline app/templates/ui/app/detail/detail.html]: changed to CSS implementation

Comment (jenbreese) [inline app/templates/ui/app/detail/detail.html]: I will look at this shortly

**Comment by jenbreese** [comment]
@grtjn and @janmichaelyu : it looks good to me. I tested it.

**Comment by grtjn** [comment]
Taking @jenbreese's word for it.. ;)

**Pull request merged**